test/integration/helper.rb

```diff
@@ -31,7 +31,7 @@ def root_group(os)
     end
 
     def selinux_label(backend, path = nil)
-      return nil if backend.class.to_s =~ /docker/i
+      return nil if backend.class.to_s =~ /docker|kubernetes/i
```

Comment:
We are preparing https://github.com/chef/inspec/issues/1661 in Train and InSpec. kubernetes will become a platform in train. At this point, we are adding a k8s transport for the execution inside of containers only. We think `k8s-exec` is a better name for this specific implementation.

Reply:
I don't understand, that PR says "platform" would be things like RedHat or Ubuntu, in which case Kubernetes will not be a platform. The OS image used in the underlying container would be the platform, just like with Docker.